ccm_web/server/src/api/api.utils.ts

```diff
@@ -20,13 +20,43 @@ export enum HttpMethod {
   Delete = 'DELETE'
 }
 
+export function checkForUniqueIdError (error: unknown): boolean {
```

Comment:
I would name this differently…

```suggestion
export function isUniqueIdError (error: unknown): boolean {
```

Because it would make its usage read more fluently as `if (isUniqueIdError(error)) return false`.

I won't merge this PR ATM, just in case you decide to rename it.  If you think it's fine as it is, tell me, then I'll proceed with merging.

Reply:
I'm just hesitating because the naming you are using is typically used for type guards, which this is not (thought it uses one).

Reply:
Makes sense.  We'll leave it as it is.